homeassistant/components/media_player/yamaha.py

```diff
@@ -25,9 +25,11 @@
 CONF_SOURCE_IGNORE = 'source_ignore'
 
 DEFAULT_NAME = 'Yamaha Receiver'
+DEFAULT_HOST = 'DISCO'
```

Comment:
Okay. There's still a lot to learn for me in Python, since i'm more used to Ruby. I have implemented the proper check in https://github.com/home-assistant/home-assistant/pull/3451/commits/8d3c4694b2368bb69cb4a7757a648f4cc87cc6b6